### PR TITLE
#14 Add code of conduct

### PR DESCRIPTION
**Pull request by AzisK**
Please get acquainted to our Contributor Covenant Code of Conduct

### CODE REVIEW

**Review by JustasBelevicius — APPROVED** [review]
Well done!

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[azuolas dot krusna at yahoo dot com].
```
Comment (rikius) [inline CODE_OF_CONDUCT.md:64]: Do we want to keep your email in this maybe it's better to have Lina email or kayak general?

Comment (AzisK) [reply CODE_OF_CONDUCT.md:64]: I will discuss with Lina about it

**Review by psaulenas — APPROVED** [review]
Good job!

**Review by rikius — APPROVED** [review]

**Review by ignasposka — APPROVED** [review]

**Comment by AzisK** [comment]
Merging as approved. Later I will add one more point in code of conduct that there could be no "religious attacks" as pointed out by Lina. So extend this part "Trolling, insulting or derogatory comments, and personal or political attacks"

**Pull request merged**